ios-slave-machine-setup.md

```diff
@@ -0,0 +1,76 @@
+# Setting up iOS slave machine. 
+
+## Manual configurations
+
+*  Disable sleep mode
+```
+# System Preferences > Energy Saver > set Turn display off after to Never
+```
+* Disable screensaver
+```
+# System Preferences > Desktop & Screen Saver > set Start after to Never
+```
+* Allow Accessibility for Terminal in order to allow mocking simulator's location for UI testing
+```
+# System Preferences > Security & Privacy > Privacy > Accessibility > Add Terminal by tapping + button
+```
+
+## Manual setup
```

Comment:
As discussed, I still think **Manual setup** and **Automatable setup** might confuse others 🤔 How about

> ## Prerequisite
> - **Xcode** - some details to be provided.
> ## Setup / How to set up / Installation
> 1. Run the following command
> ```
> $ sudo sh ./setup-ios-slave-machine.sh
> ```
> 2. Everything is all set 🎉 

or even shorter

> ## Setup / How to set up / Installation
> 1. Download and install __Xcode__
> 2. Run the following command
> ```
> $ sudo sh ./setup-ios-slave-machine.sh
> ```
> 3. Everything is all set 🎉